package.json

```diff
@@ -4,6 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "format": "prettier --check .",
+    "format:fix": "prettier --write .",
```

Comment:
I noticed we have prettier installed for the project, so I added two scripts for it.

Reply:
Moved to a different PR